shell/components/AppModal.vue

```diff
@@ -98,8 +97,22 @@ export default defineComponent({
      */
     focusTrapWatcherBasedVariable: {
       type:    Boolean,
-      default: false,
+      default: undefined,
+    },
+    fakeWatchVar: {
```

Comment:
What is the purpose of `fakeWatchVar`? It isn't immediately clear to me how this is used. Can we rename this prop so that its purpose is more clear?